gapic/src/main/com/google/gapid/util/Colors.java

```diff
@@ -26,6 +26,8 @@
   public static final int DARK_LUMINANCE8_THRESHOLD = 165;
   public static final RGB BLACK = new RGB(0, 0, 0);
   public static final RGB WHITE = new RGB(255, 255, 255);
+  public static final RGBA BLACK_RGBA = new RGBA(0, 0, 0, 255);
```

Comment:
AGI's color management system has moved to perfetto/views/StyleConstants.java, please do this part there.

Reply:
Done.